modules/op-alternative-approaches-compute-resource-quota-pipelines.adoc

```diff
@@ -30,7 +30,7 @@ spec:
 
 * Set resource limits by specifying values for the `LimitRange` object. For more information on `LimitRange`, refer to link:https://docs.openshift.com/container-platform/latest/nodes/clusters/nodes-cluster-limit-ranges.html#nodes-cluster-limit-ranges[Restrict resource consumption with limit ranges].
 
-* xref:../pipelines/reducing-pipelines-resource-consumption.adoc#reducing-pipelines-resource-consumption[Reduce pipeline resource consumption].
+* xref:../resource/reducing-pipelines-resource-consumption.adoc#reducing-pipelines-resource-consumption[Reduce pipeline resource consumption].
```

Comment:
@mramendi No xrefs in modules. Refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#links-hyperlinks-and-cross-references

Reply:
See above - it was there before the restructure cnd changing it is out of scope.